src/taxman.py

```diff
@@ -475,6 +477,54 @@ def _evaluate_taxation_GERMANY(self, op: tr.Operation) -> None:
                 )
                 self.tax_report_entries.append(report_entry)
 
+        elif isinstance(op, tr.MarginFee):
+            # Fees for margin trading
+            self.remove_from_balance(op)
+            if in_tax_year(op):
+                taxation_type = "Kapitaleinkünfte aus Margin Trading"
+                report_entry = tr.MarginReportEntry(
+                    platform=op.platform,
+                    amount=op.change,
```

Comment:
Shouldn't this be negative as we are giving it aways? Could be good to assert the `fee` value in `book.py`.

Reply:
Yes, is fixed now.
I added an assertion to all three operation types.